packages/base-controller/CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.0.2]
+
+### Changed
+
+- Bump TypeScript version to `~4.9.5` ([#4084](https://github.com/MetaMask/core/pull/4084))
+
+### Fixed
+
+- Fix misnamed parameter name ([#4108](https://github.com/MetaMask/core/pull/4108))
```

Comment:
This is an internal parameter, so I think we can omit it from the changelog. It doesn't impact the API at all